pkg/template/master_template.go

```diff
@@ -293,6 +293,7 @@ systemd:
       Description=k8s-kubelet
       StartLimitIntervalSec=0
       [Service]
+	  User=root
```

Comment:
we don't need an explicit user. It is running as root already.

Reply:
Ok, let's test it without explicit user. I based this on the following thread https://github.com/kubernetes/kubernetes/issues/45487#issuecomment-464516064